### PR TITLE
Added getDefaultTerm function to termData

### LINKED ISSUE #382: Create a `getDefaultTerm()` function for `termData`
Currently, the expression `termData[defaultTerm]` is used to access the default term, which is not the most concise.

Create a function `getDefaultTerm()` in [`termData`](https://github.com/icssc/AntAlmanac/blob/main/src/termData.js) which returns that expression, and make sure all usages of the expression are replaced with this function call. Optionally, also make sure `defaultTerm` is no longer accessible from outside of `termData`.

### PR DESCRIPTION
**Pull request by serisummer**
## Summary
Added getDefaultTerm function to termData.
Updated the imports of termData with the addition of getDefaultTerm.
Changed the usage of termData[defaultTerm] to getDefaultTerm().

## Test Plan
The function getDefaultTerm printed correct result in the console.
Screenshot 2022-06-10 143000

## Issues
Closes #382

## Future Followup  
Might need change to the way termData is accessed so that defaultTerm is not longer accessible outside of termData.

### CODE REVIEW

**Comment by ChaseC99** [comment]
> ### Future Followup
> Might need change to the way termData is accessed so that defaultTerm is not longer accessible outside of termData.

Eh that's probably fine for `defaultTerm` to still be accessible.

File: src/components/SectionTable/SectionTableBody.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import { ColorAndDelete, ScheduleAddCell } from './SectionTableButtons';
 import classNames from 'classnames';
 import { clickToCopy, isDarkMode } from '../../helpers';
 import analyticsEnum, { logAnalytics } from '../../analytics';
-import { defaultTerm, termData } from '../../termData';
+import { defaultTerm, termData, getDefaultTerm } from '../../termData';
```
Comment (ChaseC99) [inline src/components/SectionTable/SectionTableBody.js]:
```suggestion
import { getDefaultTerm } from '../../termData';
```

Unused import

File: src/stores/RightPaneStore.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
 import { EventEmitter } from 'events';
 import dispatcher from '../dispatcher';
-import { termData, defaultTerm } from '../termData';
+import { termData, defaultTerm, getDefaultTerm } from '../termData';
```
Comment (ChaseC99) [inline src/stores/RightPaneStore.js]:
```suggestion
import { getDefaultTerm } from '../termData';
```

Unused import

File: src/components/EnrollmentGraph/GraphModalContent.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import { withStyles } from '@material-ui/core/styles';
 import { FormControl, InputLabel, MenuItem, Paper, Select, Typography } from '@material-ui/core';
 import GraphRenderPane from './GraphRenderPane';
 import { queryWebsoc } from '../../helpers';
-import { termData, defaultTerm } from '../../termData';
+import { termData, defaultTerm, getDefaultTerm } from '../../termData';
```
Comment (ChaseC99) [inline src/components/EnrollmentGraph/GraphModalContent.js]:
```suggestion
import { termData, defaultTerm } from '../../termData';
```

Unused import

**Review by ChaseC99 — CHANGES_REQUESTED** [review]
Looking solid! Just remove some unused imports and it should be good to go.

**Review by ecxyzzy — APPROVED** [review]
Looks good to me—all of the changes have been addressed. Great work!

Just going to re-request a review from Chase so we can merge this.

**Review by ChaseC99 — APPROVED** [review]

**Pull request merged**